lib/components/layout/layoutOneThird.js

```diff
@@ -8,10 +8,10 @@ const LayoutOneThird = ({ left, right }) => {
   return (
     <div className={styles.wrap}>
       <Row gutter={{ xs: 16, sm: 24, md: 42 }}>
-        <Col span={16} xs={24} sm={12} md={16}>
+        <Col span={16} xs={24} sm={12} md={14}>
```

Comment:
Is this to fix this buttons not staying in their container? Will this also be okay in Secure Share since this is shared with that code as well.

If this is for the buttons should we instead change how those buttons act instead of this major layout component that will be shared by many apps? This makes the component not actually describe what it does (`LayoutOneThird` should have the side be one-third).

Reply:
With your update it is the same issue though. This is a shared component that no longer does what it says. If the content is spilling over, then we should instead fix the content, not our layout component.

Reply:
I removed this change, I'll ask how it should look on tablet.